src/test/java/dk/cngroup/lentils/service/GameLogicServiceIntegrationTest.java

```diff
@@ -28,24 +31,61 @@
     private StatusService statusService;
     @Autowired
     private GameLogicService gameLogicService;
+    @Autowired
+    private FinalPlaceService finalPlaceService;
 
     @Test
-    public void shouldHavePasedAllCyphers() {
-        Team team = new Team("team", 4, "1111");
-        teamService.save(team);
-        Cypher cypher1 = new Cypher("first", 1, new Point(8,8), "yes");
-        Cypher cypher2 = new Cypher("second", 2, new Point(8,8), "yees");
-        cypherService.save(cypher1);
-        cypherService.save(cypher2);
-        statusService.initializeStatusForTeamAndCypher(cypher1, team);
-        statusService.initializeStatusForTeamAndCypher(cypher2, team);
-        statusService.markCypher(cypher1, team, CypherStatus.SOLVED);
-        statusService.markCypher(cypher2, team, CypherStatus.SKIPPED);
+    public void shouldHavePassedAllCyphers() {
+        Team team = setCyphersToSpecificStatusesForTeam(CypherStatus.SOLVED, CypherStatus.SKIPPED);
         assertTrue(gameLogicService.passedAllCyphers(team));
     }
 
     @Test
-    public void shouldNotHavePasedAllCyphers() {
+    public void shouldNotHavePassedAllCyphers() {
+        Team team = setCyphersToSpecificStatusesForTeam(CypherStatus.SOLVED, CypherStatus.PENDING);
+        assertFalse(gameLogicService.passedAllCyphers(team));
+    }
+
+    @Test
+    public void passedTimeToViewFinalPlaceIsFalseWithNoFinalPlaceTest() {
```

Comment:
Chtelo by to i nejaky test na to, kdy metoda `passedTimeToViewFinalPlace`  vrati neco jineho nez false.

Reply:
doplneno